neurokit2/ecg/ecg_simulate.py

```diff
@@ -2,13 +2,22 @@
 import math
 
 import numpy as np
+import pandas as pd
 import scipy
 
 from ..signal import signal_distort, signal_resample
 
 
 def ecg_simulate(
-    duration=10, length=None, sampling_rate=1000, noise=0.01, heart_rate=70, heart_rate_std=1, method="ecgsyn", random_state=None
+    duration=10,
+    length=None,
```

Comment:
As a cardiologist who has done a lot of work with ECG data across many vendors and collaborations in the US, I can say that the sampling rate of all the major vendors are either 250 or 500 Hz. In spirit of trying to make synthetic data as similar as possible to real data, I would recommend changing the default sampling rate to 250.

Reply:
The motivating factor for having 1000 as a default is mostly to be consistent; we use 1000 throughout the entire NeuroKit, it would be kinda strange to have a different default just for one function, so I prefer to leave it as-is as it's easy to change anyway :)